airflow/www/views.py

```diff
@@ -82,7 +83,7 @@
 
 PAGE_SIZE = conf.getint('webserver', 'page_size')
 if os.environ.get('SKIP_DAGS_PARSING') != 'True':
-    dagbag = models.DagBag(settings.DAGS_FOLDER)
+    dagbag = models.DagBag(settings.DAGS_FOLDER)  # type: Any
```

Comment:
Why Any and not `airflow.models.DagBag`?

Reply:
I agree that this *should* be type `DagBag`, but the `else` block below defines this variable as the class `DagBag`, not an instance, which `mypy` reasonably complains about. I'm actually not sure why (or if) this works. Maybe instead of setting `dagbag` to be a class, we should pass `skip_dag_parsing` to the `DagBag` constructor so that this variable is always an instance. What do you think?

Reply:
Oh I suspect that this only "works" because this flagis only set in very few cases and most of the code path isn't exercised when it is set.

How about making the else be `dagbag = models.DagBag(os.devnull, include_examples=False)` to make it load no DAGs?